packages/compass-query-bar/src/components/query-option.tsx

```diff
@@ -198,7 +198,7 @@ const QueryOption: React.FunctionComponent<QueryOptionProps> = ({
               <TextInput
                 aria-labelledby={`query-bar-option-input-${name}-label`}
                 id={id}
-                data-testid="query-bar-option-input"
+                data-testid={`query-bar-option-${name}-input`}
```

Comment:
We somehow lost this in a refactor at some point and every query bar option input had the same data-testid 😬

Reply:
Oh no, but like at the same time none of the tests were using this I guess? Otherwise I'd expect a lot of things to blow up in tests 😓

Reply:
Yeah that's the thing - we were using the selectors without `-input`, ie. the containing elements ;)